pom.xml

```diff
@@ -204,6 +204,10 @@
 			<properties>
 				<skipTests>true</skipTests>
 				<skipITs>true</skipITs>
+				<formatter.skip>true</formatter.skip>
+				<impsort.skip>true</impsort.skip>
+				<xml-format.skip>true</xml-format.skip>
+				<japicmp.skip>true</japicmp.skip>
 			</properties>
 		</profile>
```

Comment:
@jeenbroekstra What is your opinion on skipping the formatters and japicmp in the quick profile? 

I want to speed up the build a bit on my local machine when I need to use `mvn clean install` for sharing my local RDF4J changes with another maven project. I usually also try to remember to use the parallel build (-T 1C), which makes things even faster. Some of the formatters don't support parallel builds (dunno if it's just a lack of explicit support or if something is actually broken, I just got a bunch of warnings). 

For doing a full build there is still `mvn clean install -DskipTests` that does all the formatting and also stops with an error if the api changes are breaking. Since `-DskipTests` does pretty much what the `quick` profile does today I was thinking that maybe there is room to differentiatet it by also skipping formatting and japicmp.

Reply:
Makes total sense to me, it's more or less the whole point of that profile, I just never got around to updating it. Good call!